src/app/tabs/run-tab.js

```diff
@@ -581,6 +584,53 @@ function settings (container, self) {
       }
     })
   }
+  function signMessage (event) {
+    self._deps.udapp.getAccounts((err, accounts) => {
+      if (err) { addTooltip(`Cannot get account list: ${err}`) }
+      var stringifyRSV = (rsv) => {
+        var rsvJson = () => { return JSON.stringify(rsv, null, '\t') } 
+        copyToClipboard(rsvJson).onclick(event)
```

Comment:
perhaps would be more convenient copy to clipboard the raw hexadecimal string of the encoded value.

Reply:
@yann300 done